org-admin-setup.sh

```diff
@@ -1,6 +1,6 @@
 # New project setup - Run as org admin
 
-PROJECT_ID=cymbal-eats-<random number>
+PROJECT_ID=cymbal-eats-$RANDOM
```

Comment:
Great idea, @aburhan! Thank you for making this change.
We could also add userid to the name cymbal-eats-$RANDOM

<userid>-cymbal-eats-$RANDOM